neighbour-discoverer/src/main/java/no/vegvesen/ixn/federation/discoverer/NeighbourRESTFacade.java

```diff
@@ -83,10 +83,10 @@ Interchange postCapabilities(Interchange discoveringInterchange, Interchange nei
 				logger.error("Received error object from server: {}", errorDetails.toString());
 				throw new CapabilityPostException("Error in posting capabilities to neighbour " + neighbour.getName() +". Received error response: " + errorDetails.toString());
 			} catch (JsonMappingException jme) {
-				logger.error("Unable to cast error response as ErrorDetails object.");
+				logger.error("Unable to cast error response as ErrorDetails object.", jme);
 				throw new CapabilityPostException("Error in posting capabilities to neighbour " + neighbour.getName() +". Could not map server response to ErrorDetailsobject.");
 			} catch (IOException ioe) {
```

Comment:
We only need one catch with IOException because JsonMappingException extends IOException .